bluebottle/settings/base.py

```diff
@@ -532,6 +532,14 @@
     'not_fully_funded': .05
 }
 
+CURRENCIES_ENABLED = [
+    {
+        'code': 'EUR',
+        'name': 'Euro',
+        'symbol': u"\u20AC"
+    }
+]
+
```

Comment:
@ernst I took your suggestion in consideration, but in the end I think this more verbose method is slightly better than derive it from payment methods. Are you ok with this?